"generateEmail should not include the 'newline' character'" test fails

Not sure why this is a test in the first place though - why don't we expect the emails to contain newlines?

```
  1) generateEmail
       should not include the 'newline' character:

      AssertionError: expected 0 to equal -1
      + expected - actual

      -0
      +-1
```

I did some debugging and all examples contain multiple newlines, e.g.

> \nI think it’s sad that the government has decided to go back on its manifesto pledge and is no longer providing 0.7% of GNI as foreign aid.I know the government thinks we need to provide less aid because the UK has less money ourselves, but this doesn’t stack up -- it’s a percentage of GDP, so it will go down anyway.I’m writing to ask you to ensure the government reverses this decision, or at least make absolutely sure it’s temporary.\n\n\n\n\n\nYours sincerely,\nmitch mcconell\n45 hoot street worsher

--- a/tests/spec.js
+++ b/tests/spec.js
@@ -8,12 +8,14 @@ const exampleResponses = [];
 var normalizedPath = require("path").join(__dirname, "exampleResponses");
 
 fs.readdirSync(normalizedPath).forEach(function (file) {
-  exampleResponses.push(require("./exampleResponses/" + file));
+  exampleResponses.push({ filename: file, json: require("./exampleResponses/" + file) });
 });
 
-const allGeneratedEmails = exampleResponses.map(({ form_response }) => {
-  return generateEmail(form_response);
+const allGeneratedResults = exampleResponses.map(({ filename: file, json: { form_response }}) => {
+  return { filename: file, email: generateEmail(form_response) };
 });
+
+const allGeneratedEmails = allGeneratedResults.map(result => result.email);
 
 describe("/api/postcode", () => {
   it("should return expected MP details for DL6 2NJ", async () => {
@@ -69,9 +71,9 @@ describe("generateEmail", () => {
       expect(res.body.search(/athiest/)).to.equal(-1);
     });
   });
-  it("should not include the 'newline' character", () => {
-    allGeneratedEmails.forEach((res) => {
-      expect(res.body.search(/\n/)).to.equal(-1);
+  it("should not include escaped 'newline' characters", () => {
+    allGeneratedResults.forEach((res) => {
+      expect(res.email.body, "In " + res.filename).not.to.contain("\\n");
     });
   });
 });

